client/src/pages/private/ParticipantView.js

```diff
@@ -524,95 +534,134 @@ export default () => {
-              nonHcapOpportunity: values.nonHcapOpportunity,
-              positionTitle: values.positionTitle,
-              positionType: values.positionType,
-              hiredDate: values.hiredDate,
-              startDate: values.startDate,
-              site: values.site,
-            });
-          }}
-          onClose={defaultOnClose}
+    {activeModalForm === 'hired' && <HireForm
+      sites={sites}
+      initialValues={{
+        nonHcapOpportunity: false,
+          positionTitle: '',
+          positionType: '',
+          hiredDate: '',
+          startDate: '',
+          site: '',
+          acknowledge: false,
+      }}
+      validationSchema={HireFormSchema}
+      onSubmit={(values) => {
+        handleEngage(actionMenuParticipant.id, 'hired', {
+          nonHcapOpportunity: values.nonHcapOpportunity,
+          positionTitle: values.positionTitle,
+          positionType: values.positionType,
+          hiredDate: values.hiredDate,
+          startDate: values.startDate,
+          site: values.site,
+        });
+      }}
+      onClose={defaultOnClose}
         />}
-        {activeModalForm === 'new-participant' && <NewParticipantForm
-          sites={sites}
-          initialValues={{
-            nonHcapOpportunity: false,
-            positionTitle: '',
-            positionType: '',
-            offerDate: '',
-            startDate: '',
-            site: '',
-            acknowledge: false,
-          }}
-          validationSchema={HiredParticipantSchema}
-          onSubmit={(values) => {
-            handleExternalHire({
-              firstName: values.firstName,
-              lastName: values.lastName,
-              phoneNumber: values.phoneNumber,
-              emailAddress: values.emailAddress,
-              origin: values.origin,
-              nonHcapOpportunity: values.nonHcapOpportunity,
-              contactedDate: values.contactedDate,
-              offerDate: values.offerDate,
-              startDate: values.startDate,
-              site: values.site,
+    {activeModalForm === 'edit-participant' && <EditParticipantForm
+      initialValues={{
+        ...actionMenuParticipant
+      }}
+      validationSchema={EditParticipantFormSchema}
+      onSubmit={async (values) => {
+        const history = {
+          timestamp: new Date(),
+          changes: [],
+        };
+        Object.keys(values).forEach(key => {
+          if (values[key] !== actionMenuParticipant[key]) {
+            history.changes.push({
+              field: key,
+              from: actionMenuParticipant[key],
+              to: values[key],
             });
-          }}
-          onClose={defaultOnClose}
-        />}
-      </Dialog>
+          }
+        });
+        values.history = (actionMenuParticipant.history)? [history, ...actionMenuParticipant.history] : [history];
+        const response = await fetch('/api/v1/participant', {
+          method: 'POST',
```

Comment:
This will also need to change if we swap to put/patch - just flagging to make it easy to find